src/System.Private.CoreLib/shared/System/IO/MemoryStream.cs

```diff
@@ -547,6 +547,51 @@ public override Task CopyToAsync(Stream destination, int bufferSize, Cancellatio
             }
         }
 
+        public override void CopyTo(Buffers.ReadOnlySpanAction<byte, object?> callback, object? state, int bufferSize)
```

Comment:
Nit: please add a `using System.Buffers;` at the top and then here don't use the namespace.

Reply:
Fixed.